exporters/deploytime/app.py

```diff
@@ -37,39 +38,41 @@ def collect(self) -> Iterable[GaugeMetricFamily]:
             "Deployment timestamp",
             labels=["namespace", "app", "image_sha"],
         )
-        deployment_active_metric = GaugeMetricFamily(
-            "deployment_active",
-            "Active deployments in cluster",
-            labels=["namespace", "app", "image_sha"],
-        )
+
+        number_of_dropped = 0
 
         for m in metrics:
+            if not is_out_of_date(str(m.deploy_time_timestamp)):
```

Comment:
adding this to code without adding also to docs doesn't feel good for me

Reply:
added doc to cover it.